### PR TITLE
5.1 release

### PR DESCRIPTION
**Pull request by cassidyjames**
Prepping the site for the 5.1 release

### CODE REVIEW

File: docs/learning-the-basics.md [hunk 1]
```diff
@@ -95,37 +104,40 @@ By default, all app windows open on one workspace. However, you can use multiple
...
-
-For more information related to security patches, read the <a href="https://usn.ubuntu.com/">Ubuntu security notices</a>.
+AppCenter includes updates for apps and the operating system itself under the **Installed** tab. Remember to install updates frequently to get the latest features, fixes, and security improvements. For more information related to security patches, read the [Ubuntu security notices](https://usn.ubuntu.com/).
 
 ## System Settings {#system-settings}
 
-elementary OS comes with a handy app called "System Settings" that controls all of your system-wide (or "global") preferences. System Settings gives you the ability to adjust things like keyboard shortcuts, display resolution, your wallpaper, and more.
+elementary OS comes with the "System Settings" app that lets you manage settings and preferences for your device. System Settings gives you the ability to adjust things like keyboard shortcuts, display resolution, your wallpaper, and more.
```
Comment (cassidyjames) [inline docs/learning-the-basics.md]:
```suggestion
elementary OS comes with the "System Settings" app that helps you manage settings and preferences for your device. System Settings gives you the ability to adjust things like keyboard shortcuts, display resolution, your wallpaper, and more.
```

**Comment by cassidyjames** [comment]
:rotating_light: **NOT TO BE MERGED** until we have the new ISO up and ready to go. :rotating_light: 

But it's ready for review. :smile:

File: press.php [hunk 2]
```diff
@@ -67,19 +67,19 @@
...
     </div>
 </div>
 
 <div class="grid">
     <div class="third">
         <h3>News &amp; Announcements</h3>
-        <p>We share frequent updates on development, major announcements, tips for developers, featured apps, and other new content via our Medium publication.</p>
-        <a class="button" href="https://medium.com/elementaryos" target="_blank" rel="noopener">Visit Medium</a>
+        <p>We share frequent updates on development, major announcements, tips for developers, featured apps, and other new content via our official blog.</p>
+        <a class="button" href="https://blog.elementary.io/" target="_blank" rel="noopener">Visit Blog</a>
```
Comment (danrabbit) [inline press.php]:
```suggestion
        <a class="button" href="https://blog.elementary.io/" target="_blank" rel="noopener">Visit Our Blog</a>
```

"Visit Blog" seems weird to me

File: index.php [hunk 3]
```diff
@@ -453,11 +453,12 @@
...
                     </h2>
-                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Time Limits</h4>
+                    <h4>Implement parental controls or manage your own digital wellbeing.</h4>
+                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h3>Screen Time</h3>
                     <p>Set per-user time limits for weekdays, weekends, or both.</p>
-                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h4>Internet Use</h4>
+                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h3>Internet Use</h3>
                     <p>Manage allowed websites. Rules affect all apps for the user, even if they use a different web browser.</p>
-                    <?php include('images/icons/actions/symbolic/view-grid-symbolic.svg'); ?><h4>Manage Apps</h4>
+                    <?php include('images/icons/actions/symbolic/view-grid-symbolic.svg'); ?><h3>Manage Apps</h3>
```
Comment (danrabbit) [inline index.php]: This doesn't seem right. It's inconsistent with the Applications Menu section above. Also, changes to Parental Controls haven't been released. I think there's some issue with master, but you can't do anything with your own user on a clean install right now

File: index.php [hunk 4]
```diff
@@ -453,11 +453,12 @@
                         <img src="images/icons/categories/64/preferences-system-parental-controls.svg" alt="Icon of an adult holding hand the hand of a child"/>
                         Parental Controls
                     </h2>
-                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Time Limits</h4>
+                    <h4>Implement parental controls or manage your own digital wellbeing.</h4>
+                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h3>Screen Time</h3>
                     <p>Set per-user time limits for weekdays, weekends, or both.</p>
-                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h4>Internet Use</h4>
+                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h3>Internet Use</h3>
```
Comment (cassidyjames) [inline index.php]:
```suggestion
                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h4>Internet Use</h4>
```

File: index.php (hunk 3 above)
Comment (cassidyjames) [inline index.php]:
```suggestion
                    <?php include('images/icons/actions/symbolic/view-grid-symbolic.svg'); ?><h4>Manage Apps</h4>
```

File: index.php [hunk 5]
```diff
@@ -453,11 +453,12 @@
                         <img src="images/icons/categories/64/preferences-system-parental-controls.svg" alt="Icon of an adult holding hand the hand of a child"/>
                         Parental Controls
                     </h2>
-                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Time Limits</h4>
+                    <h4>Implement parental controls or manage your own digital wellbeing.</h4>
```
Comment (cassidyjames) [inline index.php]:
```suggestion
```

File: index.php [hunk 6]
```diff
@@ -453,11 +453,12 @@
...
+                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h3>Screen Time</h3>
```
Comment (cassidyjames) [inline index.php]: We don't actually say "Time Limits" in the UI, so I'm leaving this terminology change :)

```suggestion
                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Screen Time</h4>
```

File: index.php (hunk 3 above)
Comment (cassidyjames) [reply index.php]: @danrabbit addressed :)

**Pull request merged**